eth/downloader/downloader_test.go

```diff
@@ -1053,225 +840,12 @@ func testForkedSyncProgress(t *testing.T, protocol uint, mode SyncMode) {
 	})
 }
 
-// Tests that if synchronisation is aborted due to some failure, then the progress
```

Comment:
Removed `FailedSyncProgress*` tests.  Post-merge, there are two situations where a downloader cycle will fail:
* manual head reversion during the sync cycle via `setHead` (`TestForkSyncedProgress` can be adapted to reflect this case).
* The sync downloads a seemingly-correct header chain which fails to insert into the `LightChain` (consensus bug).